tests/test_serialize.cc

```diff
@@ -136,6 +136,29 @@ struct LDLT : public SerializableType<Eigen::SerializableLDLT> {
   };
 };
 
+struct ExplainedCovarianceRepresentation
+    : public SerializableType<ExplainedCovariance> {
+  Eigen::Index n = 3;
+
+  RepresentationType create() const override {
```

Comment:
Does Eigen::MatrixXd::Random()not return an Eigen::MatrixXd?
If it's having trouble, why not skip using "auto " instead?

Reply:
Yeah I could probably just skp auto.  I'll give that a shot.